src/compat/posix/proc/exit/exit.c

```diff
@@ -33,7 +33,14 @@ void _exit(int status) {
 	panic("Returning from _exit");
 }
 
+/* from atexit */
+void __attribute__((weak)) __atexit_funcs(void)
```

Comment:
why do you want to use the weak function there?

Reply:
indeed, good idea. changed